.drone.yml

```
@@ -0,0 +1,20 @@
+---
+kind: pipeline
+type: docker
+name: committed-ci:1.4.0
```

Probably doesn't need the version

```suggestion
name: committed-ci
```